### DISCUSSION TITLE
How to Extract YAML Front Matter Data Using Markdown?

### DISCUSSION THREAD

**Discussion by amirfarzamnia** [markedjs/marked#3300] [Q&A] [answered]
I'm working with a .md file that includes YAML front matter data. Here's a snippet:

```markdown
---
description: Learn what a chatbot is, how it functions, and how to create a chatbot without coding for free.
author: amirfarzamnia
---
```

Now, how can I extract the value of "description," for example?

**Comment by UziTech** [comment, marked as answer]
See https://github.com/markedjs/marked/discussions/3290#discussioncomment-9427563

**Reply by amirfarzamnia** [reply]
Thanks, but is there a built-in method that doesn't require external modules?

**Reply by UziTech** [reply]
Nope because it isn't part of any markdown spec

**Reply by amirfarzamnia** [reply]
Thanks.
